core-plugins/pom.xml

```diff
@@ -173,6 +173,12 @@
       <artifactId>elasticmq-core_2.10</artifactId>
       <version>0.6.3</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.amazonaws/amazon-kinesis-connectors -->
+    <dependency>
```

Comment:
how much larger does this dependency make the core-plugins JAR?
How large is the amazon-kinesis-connectors jar(s)?